spec/factories/job_runs.rb

```diff
@@ -11,6 +10,7 @@
 
     trait :discovery_report do
       job_type { 'discovery_report' }
+      output_location { '/path/to/report' }
     end
   end
```

Comment:
this just makes the factory data better reflect reality (pre-assembly jobs no longer have anything in the `output_location` field)